### PR TITLE
testmap: Move lorax rhel8-branch testing to RHEL 8.4

### PR DESCRIPTION
**Pull request by atodorov**
and adjust which test scenarios are executed due to the switch to
osbuild-composer backend.

 - [x] Fix lorax tests for RHEL 8.4: https://github.com/weldr/lorax/pull/1109

### CODE REVIEW

File: task/testmap.py [hunk 1]
```diff
@@ -85,18 +85,12 @@
...
-            'rhel-8-3',
-            'rhel-8-3/live-iso',
-            'rhel-8-3/qcow2',
-            'rhel-8-3/aws',
-            'rhel-8-3/azure',
-            'rhel-8-3/openstack',
-            'rhel-8-3/vmware',
-            'rhel-8-3/tar',
-            'rhel-8-3/ci',
+            'rhel-8-4/lorax',
```
Comment (atodorov) [inline task/testmap.py:92]: @bcl from what I can see lorax' `test/run` doesn't respect the `ci` scenari anymore but does respect a `lorax` scenario so I've added that both for master and rhel8-branch branches.

**Comment by martinpitt** [comment]
I triggered the "rhel-8-4" scenario, but as our CI  does not currently know about the "rhel-8-4/lorax" scenario, I can't trigger it yet. This needs to go into `_manual` before. I'll look at it after lunch, I can fake this for the time being.

**Comment by martinpitt** [comment]
@atodorov : Hmm, now you dropped the "rhel-8-4" scenario as well, was that on purpose?

**Comment by martinpitt** [comment]
[/osbuild-composer scenario failed](https://logs.cockpit-project.org/logs/pull-1587-20210126-121430-0f45521e-rhel-8-4-osbuild-composer-weldr-lorax-rhel8-branch/log.html)

**Comment by atodorov** [comment]
> @atodorov : Hmm, now you dropped the "rhel-8-4" scenario as well, was that on purpose?


Yes. At present `test/run` in lorax responds only to lorax or osbuild-composer scenarios. It will fall back to a default execution if scenario is empty but it won't setup the environment properly.

**Comment by atodorov** [comment]
> [/osbuild-composer scenario failed](https://logs.cockpit-project.org/logs/pull-1587-20210126-121430-0f45521e-rhel-8-4-osbuild-composer-weldr-lorax-rhel8-branch/log.html)

This is being dealt with in https://github.com/weldr/lorax/pull/1109.  Let me know which one you want to merge first b/c the lorax counterpart is not ready yet.

**Comment by martinpitt** [comment]
@atodorov Thanks! Let's please land the fix first, otherwise we'll incur kwon-broken tests to every unrelated lorax PR and image refresh.

**Comment by atodorov** [comment]
@martinpitt I've merged the changes on the lorax side, can you kick off the test jobs again b/c I don't have sufficient permissions on this repository to do so.

**Comment by martinpitt** [comment]
@atodorov : Appreciated, thanks! Kicked the tests

**Review by martinpitt — APPROVED** [review]

**Pull request merged**